usecase/usecase-webapi-xwalk-tests/tests.full.xml

```diff
@@ -1,111 +1,213 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <test_definition>
   <suite name="Usecase WebAPI">
-    <set name="Runtime &amp;amp; Packaging" background="brand-primary" icon="glyphicon-asterisk">
-      <testcase component="sample" execution_type="manual" id="AppURI" platform="all" priority="P0" purpose="App URI" status="approved" type="functional_positive">
+    <set name="Runtime &amp;amp; Packaging" background="brand-primary" icon="glyphicon-asterisk" type="script">
+      <testcase component="sample" execution_type="auto" id="AppURI" platform="all" priority="P0" purpose="App URI" status="approved" type="functional_positive">
         <description>
-          <test_script_entry test_script_expected_result="0">/opt/usecase-webapi-xwalk-tests/bddrunner testscripts/AppURI.feature</test_script_entry>
```

Comment:
where is the ui-auto attribute? Please keep tests.full.xml as a comprehensive version, sub tests.xml should be converted from it.